pkg/k8s/watchers/pod.go

```diff
@@ -665,6 +672,15 @@ func (k *K8sWatcher) deleteHostPortMapping(pod *slim_corev1.Pod, podIPs []string
 }
 
 func (k *K8sWatcher) updatePodHostData(oldPod, newPod *slim_corev1.Pod, oldPodIPs, newPodIPs k8sTypes.IPSlice) error {
+	logger := log.WithFields(logrus.Fields{
+		logfields.K8sPodName:   newPod.ObjectMeta.Name,
+		logfields.K8sNamespace: newPod.ObjectMeta.Namespace,
+	})
+	if newPod.Spec.HostNetwork {
+		logger.Debug("Pod is using host networking")
```

Comment:
nit: We don't need to initialize the logger outside the case where it's used. But also I don't anticipate that this is a super hot path, so this shouldn't be too costly. :shrug: Probably fine as-is but worth considering in future if you add new debug statements somewhere that is hit frequently.